packages/terra-clinical-item-collection/src/_ItemCollectionUtils.jsx

```diff
@@ -103,7 +99,7 @@ function createOnSelectProps(onSelect, key) {
   };
 
   selectableProps.onKeyDown = (event) => {
-    if (event.nativeEvent.keyCode === KEYCODES.ENTER || event.nativeEvent.keyCode === KEYCODES.SPACE) {
+    if (event.nativeEvent.keyCode === KeyCode.KEY_ENTER || event.nativeEvent.keyCode === KeyCode.KEY_SPACE) {
```

Comment:
Should be KEY_RETURN here too.

Reply:
Here is my fix. https://github.com/cerner/terra-clinical/pull/479/commits/5fad38da91527764167715a3cfc8bbe5174bc3b5#diff-c12336a4490f4dce5ed72463a37231f3R102